libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -809,15 +844,19 @@ def _coerce_string(value: Any) -> str:
 
     def _extract_where_clause_cql(
         self,
-        metadata_keys: list[str] = [],  # noqa: B006
+        has_id: bool = False,
+        metadata_keys: list[str] | None = None,
```

Comment:
Could also use `metadata_keys: Sequence[str] = ()` and not have to do the None-check